composer.json

```diff
@@ -27,25 +27,26 @@
         "league/commonmark": "^1.5",
         "nyholm/psr7": "^1.2",
         "oskarstark/env-var-extension": "^1.0",
-        "symfony/cache": "^5.3",
-        "symfony/console": "^5.3",
-        "symfony/dotenv": "^5.3",
+        "symfony/cache": "^5.4",
+        "symfony/console": "^5.4",
+        "symfony/dotenv": "^5.4",
         "symfony/flex": "^1.10",
-        "symfony/framework-bundle": "^5.3",
-        "symfony/http-client": "^5.3",
+        "symfony/framework-bundle": "^5.4",
+        "symfony/http-client": "^5.4",
         "symfony/monolog-bundle": "^3.6",
-        "symfony/notifier": "^5.3",
-        "symfony/slack-notifier": "^5.3",
-        "symfony/string": "^5.3",
-        "symfony/twig-bundle": "^5.3",
-        "symfony/yaml": "^5.3",
+        "symfony/notifier": "^5.4",
+        "symfony/slack-notifier": "^5.4",
+        "symfony/string": "^5.4",
+        "symfony/twig-bundle": "^5.4",
+        "symfony/yaml": "^5.4",
         "twig/extra-bundle": "^3.1",
         "twig/markdown-extra": "^3.1",
         "twig/twig": "^2.12 || ^3.0",
         "webmozart/assert": "^1.9"
     },
     "require-dev": {
         "ergebnis/test-util": "^1.2",
+        "friendsofphp/php-cs-fixer": "^3.8",
```

Comment:
We should be careful with this one, it has a lot of dependencies, we should pick a minimum version that supports our whole version support (sf 4.4, 5.3, 6.0): https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/composer.json

Maybe on the dev-kit can stay with ^3.8, but we should pick ^3.4 for other Sonata Bundles (https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/composer.json) so it is possible to install sf 4.4 on the prefer lowest range.